CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ document.
 
 [3aea860...master](https://github.com/rust-lang/rust-clippy/compare/3aea860...master)
 
+* New lints:
+  * [`ifs_same_cond_fn`] [#4814](https://github.com/rust-lang/rust-clippy/pull/4814)
```

Comment:
This has to be done anyway for many more PRs, so just write the changelog in the PR body.